.github/workflows/tests.yaml

```diff
@@ -71,6 +71,8 @@ jobs:
         run: bin/run-browser-test-env --aws --ci
       - name: Run browser tests with Localstack
         run: bin/run-browser-tests-ci-batch ${{ matrix.batch_number }}
+      - name: Check no new image snapshots
+        run: git add .; git diff --exit-code HEAD
```

Comment:
How does this know to fail just on the snapshot images?

Reply:
I doesn't know that. And it fails on any diff. Given that we don't expect diffs it should be reasonable? Though I can add only `browser-test/image_snapshots` folder. More problematic though that it fails with not very readable message. Like if I saw that error I wouldn't know why it failed. Need to see if we can prettify it somehow.

Reply:
Ok, I think now it's better. As you suggested it checks only image snapshots and also uses compact diff format.